### DISCUSSION TITLE
Problem installing sll certificate

### DISCUSSION THREAD

**Discussion by kiven943** [postalserver/postal#2169] [Help with using Postal]
When I put it next
certbot certonly --webroot --webroot-path /opt/postal/app/public --domains mail.yourdomain.com

Get the next error:
/opt/postal/app/public does not exist or is not a directory

What is the problem and what is the solution?

**Comment by willpower232** [comment]
Unfortunately the website is now inside a docker container so isn't necessarily a path you or certbot can get to without some extra configuration.

You could modify the docker-compose.yml to try and share the public directory but I'm not sure how well that would work long term.

Off the top of my head, your other options could include

If you're using Caddy as your proxy to access the website, you could repurpose the certificate it has generated for you.

If you're using nginx as your proxy to access the website, you could create a special routing for the .well-known directory so it is separate from the containers.

Alternatively, you could look at using the DNS method from certbot which doesn't need any paths at all.

**Comment by kiven943** [comment]
Thank you
I was able to solve the problem and complete everything, but there is another problem that I am facing now and it is
When I want to connect smtp to any messenger, it gives me an error: Timeout or winsock error 10060
How can I fix this problem

**Reply by willpower232** [reply]
there is either a typo somewhere or a firewall in the way I'd guess, might be worth googling the specific error message you're getting in case there is a non-postal fix for it
